bodo/libs/_array.cpp

```diff
@@ -497,6 +497,13 @@ table_info* arr_info_list_to_table(array_info** arrs, int64_t n_arrs) {
     return new table_info(columns);
 }
 
+// Raw pointers since called from Python
+void append_arr_info_list_to_cpp_table(table_info* table, array_info** arrs,
+                                       int64_t n_arrs) {
+    std::vector<std::shared_ptr<array_info>> columns(arrs, arrs + n_arrs);
+    table->columns.insert(table->columns.end(), columns.begin(), columns.end());
+}
```

Comment:
The memory management of all these array pointers seems scary, but seems correct to me.

Reply:
Is there a better approach to this. All I want is to concatenate the dataframe table and index arrays together, ideally in Python

Reply:
I think this is fine at least for now. I just wanted to point out in case someone notices something.